packages/account/src/Components/account-limits/account-limits.tsx

```diff
@@ -255,11 +289,13 @@ const AccountLimits = ({
                                                 <React.Fragment>
                                                     <tr>
                                                         <AccountLimitsTableCell>
-                                                            {is_appstore ? (
-                                                                <Localize i18n_default_text='Total withdrawal limit' />
-                                                            ) : (
-                                                                <Localize i18n_default_text='Total withdrawal allowed' />
-                                                            )}
+                                                            <Localize
+                                                                i18n_default_text={
+                                                                    is_appstore
+                                                                        ? 'Total withdrawal limit'
+                                                                        : 'Total withdrawal allowed'
```

Comment:
this needs to be updated to the old way
this doesn't work on deriv-app translations